motor/providers/microsoft/platform.go

```diff
@@ -34,7 +34,40 @@ func (p *Provider) Roles() map[string]struct{} {
 	return p.rolesMap
 }
 
+func getTitleForPlatformName(name string) string {
+	switch name {
+	case "azure-compute-vm":
+		return "Azure Virtual Machine"
+	case "azure-sql-server":
+		return "Azure SQL Server"
+	case "azure-mysql-server":
+		return "Azure MySQL Server"
+	case "azure-mariadb-server":
+		return "Azure MariaDB Server"
+	case "azure-postgresql-server":
+		return "Azure PostgreSQL Server"
+	case "azure-storage-account":
+		return "Azure Storage Account"
+	case "azure-storage-container":
+		return "Azure Storage Container"
```

Comment:
Same for this one: https://github.com/mondoohq/cnquery/pull/963/files#diff-914af7db2b557306fddc6f2c387e9de21ddd9ad0868906028516c2d8d25fa76cR37